scss/_variables.scss

```diff
@@ -0,0 +1,29 @@
+$container-width: 860px;
+$medium-padding: 40px;
+
+$color-1: #eeeeef;
+$color-2: #ffffff;
+$color-3: black;
+$color-4: #384048;
+$color-5: #50c3af;
+$color-6: #787b80;
+$color-7: #d2d5da;
+
+$medium-weight: 500;
+$black-weight: 900;
+$light-weight: 300;
+
+$basic-font-stack: PTSans-Regular, AdobeInvisFont, MyriadPro-Regular;
+$heading-font-stack: PTSans-Bold, AdobeInvisFont, MyriadPro-Regular,
+  $basic-font-stack;
+
+@mixin to640 {
+  @media only screen and (max-width: 768px + 17px) {
+    @content;
+  }
+}
+@mixin to480 {
```

Comment:
not sure that using a mixing is a best thing here. What if later you will need to change breakpoint? Then you will have to rename this in all files of the project.

You could use variables like `large-break-point` and `small-break-point` and use then inside media queries in each file where you need this.

Reply:
Good point. I changed it.